extensions/ftp/pom.xml

```diff
@@ -29,11 +29,38 @@
             <groupId>commons-net</groupId>
             <artifactId>commons-net</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
In order to do that we have to expose the packages from this library but that's something that we would not do (at least for the time being). Or we could have commons-net as a plugin instead and that should go as a shared plugins /lib/opt (as core, and other mule plugins).
Still it is a conversion that needs to happen in order to understand that isolation goes against reuse, at least as jars.